app/src/main/res/layout/list_item.xml

```diff
@@ -46,6 +46,23 @@
             android:textStyle="bold" />
 
         <TextView
+            android:id="@+id/append"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="23"
+            android:textAppearance="?android:attr/textAppearanceButton"
+            android:textColor="@android:color/black"
+            android:textStyle="bold|italic" />
```

Comment:
We can now do things with the append in the UI, change text color, change style, etc. Very cool.